### ISSUE TITLE
Find best way to track community contributions

### ISSUE THREAD

**Issue by jlowin** [PrefectHQ/prefect#851]
To ensure we can recognize contributions in each release, we should track them. Perhaps a new section of the changelog? Perhaps a new file? Do we need to maintain this on an ongoing basis (each PR), or can we query github for each contributors username since a certain commit?

**Comment by cicdw** [comment]
I don't think we really need to do anything here other than track our release dates. For example, here is a CLI for extracting the number of commits by author since we open sourced on March 24:
```
git log --pretty=format:%an --since=2019-03-24 | awk '{ ++c[$0]; } END { for(cc in c) printf "%5d %s\n",c[cc],cc; }' | sort -r

   22 Jeremiah Lowin
   20 Josh Meek
    7 Chris White
    3 Ngan Hoang
    1 Dave Hirschfeld
    1 Bas Harenslak
```
This just displays everyone's name.  We could also use the [GitHub API](https://developer.github.com/v3/repos/statistics/#get-contributors-list-with-additions-deletions-and-commit-counts) to extract everyone's github username.

**Comment by jlowin** [comment]
That's great -- but more pointedly, I'd like to make sure we have our contributors listed somewhere, either in the release notes or somewhere else. What do you think is best?

**Comment by cicdw** [comment]
Ah ok gotcha; so for now right I think I want to avoid listing a name with each Changelog entry, simply because some forms of PRs (e.g., documentation typos) don't end up as Changelog entries.

However, within the Changelog we could have another new section "Contributors" and everyone can add their name to that for each release if it's not already there.  Prior to cutting a release we can always do a double check to make sure no one forgot their name.  At some point this could grow to be a long list, so we'll want to think about what to do if it gets _too_ long, but that's a good problem to have.

**Comment by cicdw** [comment]
Closed via #871
